src/index.js

```diff
@@ -6,16 +6,17 @@ import App from './App';
 
 const elements = document.getElementsByClassName('rstore-domain-search');
 Array.prototype.forEach.call(elements, element => {
+  //variable nameing is based on WordPress shortcode format
```

Comment:
Grammar nitpick: add a space after the `//`. Also, `nameing` should be `naming` 😬

`// variable naming is based on WordPress shortcode format`